packages/mobile/src/send/saga.ts

```diff
@@ -223,19 +225,22 @@ export function* buildSendTx(
   comment: string
 ) {
   const contract: Contract = yield call(getERC20TokenContract, tokenAddress)
-  const coreContract: CeloTokenWrapper<any> | undefined = yield call(
-    getTokenContractFromAddress,
-    tokenAddress
-  )
+  const coreContract: Contract = yield call(getStableTokenContract, tokenAddress)
```

Comment:
Using the contract via the abi so new stable tokens work out of the box in the future, with no need to update the code